frontend/src/lib/icons/index.ts

```diff
@@ -24,4 +24,4 @@ export {
   TrashIcon
 }
 
-export type IconString = `i-mdi-${string}`;
+export type IconString = `i-mdi-${string}` | `loading loading-spinner loading-${string}`;
```

Comment:
I just played with this a bit. Can we change it to this:
```suggestion
type DaisySize = 'xs' | 'sm' | 'md' | 'lg';
export type IconString = `i-mdi-${string}` | `loading loading-spinner${'' | ` loading-${DaisySize}`}` ;
```

Reply:
@myieye I'm not sure what happened but your suggestion is a bit screwed up. It looks like a valid value would be `loading-spinnerloading-sm`? I think it could be better pulling loading into it's own type and making that type a case for icon string.

Reply:
I think GitHub put a line break on the space after the backtick on line 28. There is a space there, so it would be `loading loading-spinner loading-(size)` or else `loading loading-spinner`. Sure, that would narrow down the types and help catch typos a bit better; I'll do it.

Reply:
Done in commit ae57e274.